log10/_httpx_utils.py

```diff
@@ -564,7 +619,8 @@ def parse_openai_responses(self, responses: list[str]):
         finish_reason = ""
 
         for r in responses:
-            if self.is_openai_response_end_reached(r):
+            # For perplexity, the last item in the responses is empty
+            if self.is_openai_response_end_reached(r) or not r:
```

Comment:
do we want to put `not r` into `is_openai_response_end_reached` function?

`return not r or "data: [DONE]" in text`